### ISSUE TITLE
Several Images are the representation of more than one object: Possible Bug

### ISSUE THREAD

**Issue by tschleider** [silknow/converter#49]
@DominicClermont made us aware that several images (106, all from MFA) appear as representation property of more than one man-made object. 

@DominicClermont  gave us the [full list](https://github.com/silknow/converter/files/4528778/multi_objects_per_image.txt) of images with their object UUIDS where this appears, so that we can investigate this issue. 

I checked [one image](http://silknow.org/silknow/media/mfa-boston/122179_0.jpg) and two object having it ([Object 1](https://data.silknow.org/object/32a48041-8a9e-327b-b13b-2ff06ff372c0) , [Object 2](https://data.silknow.org/object/d8829867-dff0-3b1f-973a-46ab9ef5f500)), but to me it seems to be a problem specific to MFA, as this is reflected in the original JSON and therefore appaently originally on their website, too.

**Comment by tschleider** [comment]
Now the MFA website works again, so it's even easier to prove that MFA themselves assigned the same picture to different objects, see for example here 

https://collections.mfa.org/objects/71055
and
https://collections.mfa.org/objects/71064
(there are five objects in total that share one of the two images)

What we have in the KG is literally how they have organised it internally: it is the same one picture with the same filename. In the KG it's https://silknow.eu/silknow/media/mfa-boston/71054_1.jpg, i've randomly chosen this example from @DominicClermont 's list . 

@rtroncy : How should we proceed?
I took this image and the one in the initial post as examples, but further checks confirm that these two cases are true for the whole list: It's MFA specific and the KG just reflects what the museum did.
